docs/Rules/ShouldProcess.md

```diff
@@ -14,55 +14,52 @@ title: ShouldProcess
 If a cmdlet declares the `SupportsShouldProcess` attribute, then it should also call
 `ShouldProcess`. A violation is any function which either declares `SupportsShouldProcess` attribute
 but makes no calls to `ShouldProcess` or it calls `ShouldProcess` but does not declare
-`SupportsShouldProcess`
+`SupportsShouldProcess`.
 
 For more information, please refer to `about_Functions_Advanced_Methods` and
-`about_Functions_CmdletBindingAttribute`
+`about_Functions_CmdletBindingAttribute`.
```

Comment:
```suggestion
For more information, please refer to [about_Functions_Advanced_Methods][1] and
[about_Functions_CmdletBindingAttribute][2].

[1]: /powershell/module/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Methods
[2]: /powershell/module/Microsoft.PowerShell.Core/About/about_Functions_CmdletBindingAttribute
```

Reply:
Thank you for your suggestion. I'm a little concerned that an absolute path may switch the language when the user is reading this page in English using [Read in English] button. I'll use `Get-Help` for these references as other rule pages.

Reply:
Please take my original suggestion. The links I provided are how our docs system works.

Reply:
Sure.

Reply:
The latest iteration with your suggestion triggered a test failure in UseCorrectCasing. Could you check if it is a real problem or a false positive mentioned in #1708?